akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -141,3 +142,16 @@ class Slf4jLoggingFilter(settings: ActorSystem.Settings, eventStream: EventStrea
   def isDebugEnabled(logClass: Class[_], logSource: String) =
     (eventStream.logLevel >= DebugLevel) && Logger(logClass, logSource).isDebugEnabled
 }
+
+/** Wraps [[org.slf4j.Marker]] */
+final class Slf4jLogMarker(marker: org.slf4j.Marker) extends LogMarker(name = marker.getName) {
```

Comment:
good point, this marker should be a val, though the usage of getting it is pretty much the exact same thing among the APIs, perhaps I should drop the "java api" and just make marker a val and be done with it :)